centreon/tests/e2e/cypress/e2e/Authentication/Web-SSO/02-web-sso-authentication-enabled.feature

```diff
@@ -1,9 +1,10 @@
+@REQ_MON-22140
 Feature: Web SSO authentication
     As an admin of Centreon Platform
     I want to be able to use an external authentication provider
     So the platform users can use an existing authentication services to authenticate
 
-@ignore-scenario
+@TEST_MON-22141-scenario
```

Comment:
should be ignored ?

Reply:
@kduret, this tag was in the feature file, i didn't want to touch it, i was afraid that there is a purpose for its existing

Reply:
@hyahiaoui-ext what is the purpose of the `@ignore-scenario` and how it works ?

Reply:
The purpose was to ignore the feature file